src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -743,25 +745,98 @@ def compute_nhd_routing_v02(
+                                        diff_inputs["dt_ub_g"],
+                                        diff_inputs["dt_db_g"],
+                                        diff_inputs["nts_ql_g"],
+                                        diff_inputs["nts_ub_g"],
+                                        diff_inputs["nts_db_g"],
+                                        diff_inputs["mxncomp_g"],
+                                        diff_inputs["nrch_g"],
+                                        np.asfortranarray(diff_inputs["z_ar_g"]),
+                                        np.asfortranarray(diff_inputs["bo_ar_g"]),
+                                        np.asfortranarray(diff_inputs["traps_ar_g"]),
+                                        np.asfortranarray(diff_inputs["tw_ar_g"]),
+                                        np.asfortranarray(diff_inputs["twcc_ar_g"]),
+                                        np.asfortranarray(diff_inputs["mann_ar_g"]),
+                                        np.asfortranarray(diff_inputs["manncc_ar_g"]),
+                                        np.asfortranarray(diff_inputs["so_ar_g"]),
+                                        np.asfortranarray(diff_inputs["dx_ar_g"]),
+                                        diff_inputs["nhincr_m_g"],
+                                        diff_inputs["nhincr_f_g"],
+                                        np.asfortranarray(diff_inputs["ufhlt_m_g"]),
+                                        np.asfortranarray(diff_inputs["ufqlt_m_g"]),
+                                        np.asfortranarray(diff_inputs["ufhlt_f_g"]),
+                                        np.asfortranarray(diff_inputs["ufqlt_f_g"]),
+                                        diff_inputs["frnw_col"],
+                                        np.asfortranarray(diff_inputs["frnw_g"].astype('double')),
+                                        np.asfortranarray(diff_inputs["qlat_g"]),
+                                        np.asfortranarray(diff_inputs["ubcd_g"]),
+                                        np.asfortranarray(diff_inputs["dbcd_g"]),
+                                        diff_inputs["cfl_g"],
+                                        diff_inputs["theta_g"],
+                                        diff_inputs["tzeq_flag_g"],
+                                        diff_inputs["y_opt_g"],
+                                        diff_inputs["so_llm_g"],
+                                        diff_inputs["ntss_ev_g"],
+                                )
+                
+                # unpack outputs to Pandas dataframe
+                dat_q_df = diff_utils.unpack_output(
+                                            diff_inputs["pynw"], 
+                                            diff_inputs["ordered_reaches"], 
+                                            out_q, 
+                                            out_elv
+                                            )
```

Comment:
Unpack outputs into Pandas DataFrame. Need to edit `unpack_output` to mimic the DataFrame structure output by `mc_reach`